compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -439,6 +439,8 @@ final class Conversions(
           optLocation.map(loc => ncBuilder.setLocation(convertLocation(loc)))
           faBuilder.setNoControllers(ncBuilder.build)
 
+        case _: FailedAuthorization.NoAuthorizers => ??? // TODO #15882
```

Comment:
Could you use the full URL for the issue.
 - It is easier to find for non-done todos if they all follow the same convention
 - quicker to find the issue. 
 - survives repo moves
 - allow to be defined in another repo. 

```suggestion
        case _: FailedAuthorization.NoAuthorizers => 
          // TODO https://github.com/digital-asset/daml/issues/15882
          ???
```

Reply:
Ok, I will address this once I get my current PR's landed.